README.md

```diff
@@ -16,268 +16,188 @@
+Ludwig makes it easy to serve deep learning models, including on GPUs. Launch a REST API for your trained Ludwig model.
 
 ```
-ludwig evaluate --dataset path/to/data.csv --model_path /path/to/model
+ludwig serve --model_path=/path/to/model
 ```
 
-This will produce evaluation performance statistics that can be visualized by the `visualize` tool, which can also be used to compare performances and predictions of different models, for instance:
+Ludwig supports exporting models to efficient Torschscript bundles.
 
 ```
-ludwig visualize --visualization compare_performance --test_statistics path/to/test_statistics_model_1.json path/to/test_statistics_model_2.json
-```
-
-will return a bar plot comparing the models on different metrics:
-
-![Performance Comparison](https://github.com/ludwig-ai/ludwig-docs/raw/master/docs/images/compare_performance.png "Performance Comparison")
-
-A handy `ludwig experiment` command that performs training and prediction one after the other is also available.
-
-## Programmatic API
-
-Ludwig also provides a simple programmatic API that allows you to train or load a model and use it to obtain predictions on new data:
-
-```python
-from ludwig.api import LudwigModel
-
-# train a model
-config = {...}
-model = LudwigModel(config)
-train_stats = model.train(training_data)
-
-# or load a model
-model = LudwigModel.load(model_path)
-
-# obtain predictions
-predictions = model.predict(test_data)
+ludwig export_torchscript -–model_path=/path/to/model
 ```
 
-`config` containing the same information of the YAML file provided to the command line interface. More details are
-provided in the [configuration documentation](https://ludwig-ai.github.io/ludwig-docs/latest/configuration/) and in
-the [API documentation](https://ludwig-ai.github.io/ludwig-docs/latest/user_guide/api/LudwigModel/).
-
-## Extensibility
+# Examples
 
-Ludwig is built from the ground up with extensibility in mind. It is easy to add
-an additional datatype that is not currently supported by adding a
-datatype-specific implementation of abstract classes that contain functions to
-preprocess the data, encode it, and decode it.
+- [Text Classification](https://ludwig-ai.github.io/ludwig-docs/latest/examples/text_classification)
+- [Named Entity Recognition Tagging](https://ludwig-ai.github.io/ludwig-docs/latest/examples/ner_tagging)
+- [Natural Language Understanding](https://ludwig-ai.github.io/ludwig-docs/latest/examples/nlu)
+- [Machine Translation](https://ludwig-ai.github.io/ludwig-docs/latest/examples/machine_translation)
+- [Chit-Chat Dialogue Modeling through seq2seq](https://ludwig-ai.github.io/ludwig-docs/latest/examples/seq2seq)
+- [Sentiment Analysis](https://ludwig-ai.github.io/ludwig-docs/latest/examples/sentiment_analysis)
+- [Image Classification](https://ludwig-ai.github.io/ludwig-docs/latest/examples/image_classification)
```

Comment:
FYI....When the PR for the reorganized table of contents for the Examples is submitted, this example will not be there.  This is basically a duplicate of the MNIST example with just different category labels.  I decided to remove it because it does not show anything different than the MNIST example.

Reply:
Removed/reorganized!